### PR TITLE
TGUI update

### PR DESCRIPTION
**Pull request by SplinterGP**
## About The Pull Request
yep, added more stuff, seeing if i can fix things and make it work woo.

## Changelog
:cl: 
fix: TGUI alerts now display buttons in a column when there's more than 2 options.
fix: TGUI alerts can now be disabled via turning TGUI inputs off in prefs.
fix: Fixed some bluescreens related to TGUI inputs.
fix: Fixes a runtime with material containers.
fix: TGUI now alerts the user when it is waiting on a refresh cooldown. In many cases, this will prevent screens from breaking that rely on static data.
fix: TGUI static data loading screens no longer resize the window temporarily. Note: If you are using a UI that gets a lot of these, please make an issue report.
fix: native fast gui boxes work again
fix: TGUI and Tooltip windows now work with high-dpi web browsers (People who are using Chromium through black magick or who have manually enabled DPI scaling for IWebBrowser2, or its already native due to byond updates ? idk)
qol: The reconnect button in tgchat will now reopen the game.
refactor: Ping system of tgui-panel has been reworked, which fixes things mentioned below.
fix: AFK players now get properly detected and kicked
fix: Automatic reconnect should now work 100% of the time
fix: Manual reconnect should now work 100% of the time
add: The accounting console has been added to the HOP's office, allowing them to track purchases, and bank balances of the crew on station. It's not illegal, it's the competitive advantage!
fix: renamed suffix for tgui-polyfill from bundle.js to min.js to prevent CBT failing after target 'clean'
code: Prettier is now recommended as an extension for UI development (or just in general!)
refactor: Many, many interfaces have been hit with the prettier stick so please report any issues. There should be zero noticeable differences in how UIs look or function.
add: Examining and other chat outputs now display in blocks to make them easier to see
fix: AI and borgs no longer have their name show twice upon being examined
fix: Fixed "Script Error" in tgui and tgchat (hopefully).
fix: Multiline inputs can now finally support line breaks, accessible via shift-enter! A maximum of two consecutive line breaks is allowed, any more than that will be automatically stripped. This only applies to input boxes that start with the multiline view enabled, and will thus not work on input boxes that aren't meant to support it.
code: restrictedinput can now accept negative numbers
qol: the full window refresh time is now one second instead of five seconds
fix: refreshing windows ("please wait...") no longer hang for up to twice as long as they should
/:cl:

### CODE REVIEW

File: code/modules/mob/living/silicon/ai/examine.dm [hunk 1]
```diff
@@ -26,6 +26,10 @@
 
 		msg += "</span>"
 
-	msg += "*---------*</span>"
+	msg += "</span>"
 
 	return list(msg)
+
+/mob/living/silicon/ai/get_examine_string(mob/user, thats)
+	null
```
Comment (SplinterGP) [inline code/modules/mob/living/silicon/ai/examine.dm:34]:
```suggestion
	return null
```

**Comment by SplinterGP** [comment]
should be good to go, tgui say stuff is in tgui folder but aint ingame, needs later porting

**Comment by TiviPlus** [comment]
Ran all test suites.
=> Finished 'tgui-test' in 11.459s
=> Finished 'tgui-tsc' in 14.564s
[warn] packages/tgui/interfaces/MarineCasship.tsx
[warn] Code style issues found in the above file(s). Forgot to run Prettier?
=> Target 'tgui-prettier' failed in 20.35s, exit code: 1
=> Finished 'tgui-eslint' in 22.51s
=> Target 'tgui-lint' failed
=> Target 'lint' failed

File: code/modules/mob/living/carbon/human/human_attackhand.dm [hunk 2]
```diff
@@ -213,6 +212,8 @@
 
 /mob/living/carbon/human/proc/check_self_for_injuries()
 	var/list/final_msg = list()
+	visible_message(span_notice("[src] examines [p_them()]self."))
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/human/human_attackhand.dm:215]: Balloon alert to viewers this

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Comment by TiviPlus** [comment]
Merging because zachary isnt responding

**Pull request merged**